src/models/UserModel.js

```diff
@@ -21,6 +21,15 @@ const userSchema = mongoose.Schema(
             type: String,
             required: true,
         },
+        email: {
```

Comment:
The email has to be between the lastname and password.
Follow this:
ci,
name,
lastname,
email,
password,
gender,
imagePath

Reply:
Fixed